server/core/gw_utils.c

```diff
@@ -61,6 +61,12 @@ setipaddress(struct in_addr *a, char *p)
 
     hint.ai_socktype = SOCK_STREAM;
 
+    if (strchr(p, '%') != NULL)
+    {
+        MXS_ERROR("Host %s contains wildcard, return", p);
```

Comment:
I think this message could be changed to `MXS_INFO` as it isn't really helpful to the end user. The INFO log level can be used when debugging connection or user problems and then this might be useful.